### PR TITLE
FUSETOOLS2-3468 - install JDK 8 and 11 required for PR tests

### PR DESCRIPTION
**Pull request by apupier**
requires https://github.com/jbosstools/jbosstools-fuse/pull/1691

### CODE REVIEW

**Comment by apupier** [comment]
there was one failign test, which is a lot better, and I think it is a flaky one:

```
2021-05-18T08:46:58.7646376Z Errors: 
2021-05-18T08:46:58.7647474Z   ProjectLocalRunTest>DefaultTest.defaultClean:110 » WaitTimeoutExpired Timeout ...
```

File: .github/workflows/build_test-PR-templates.yml [hunk 1]
```diff
@@ -27,11 +26,22 @@ jobs:
...
       with:
-        java-version: ${{ matrix.java }}
-        java-package: jdk # (jre, jdk, or jdk+fx) - defaults to jdk
+        distribution: 'adopt'
+        java-version: 8
+        java-package: jdk
+        architecture: x64
+    - name: Set JDK_8_LOCATION
+      run: |
+        echo "JDK_8_LOCATION=$JAVA_HOME" >> $GITHUB_ENV
```
Comment (lhein) [inline .github/workflows/build_test-PR-templates.yml:38]: don't u need to `export` that variable?

Comment (apupier) [reply .github/workflows/build_test-PR-templates.yml:38]: I do not think so: https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable

Comment (lhein) [reply .github/workflows/build_test-PR-templates.yml:38]: ok, good to know

**Comment by apupier** [comment]
current failure seems ot be due to a timeout:

```
2021-05-18T13:30:30.1012066Z Running org.fusesource.ide.projecttemplates.tests.integration.wizards.FuseIntegrationProjectCreatorRunnableCheckCatalogCacheLoadingIT
2021-05-18T13:30:30.9101853Z WARNING: An illegal reflective access operation has occurred
2021-05-18T13:30:30.9106427Z WARNING: Illegal reflective access by org.jboss.tools.common.jdt.debug.tools.internal.Tools (file:/home/runner/.m2/repository/p2/osgi/bundle/org.jboss.tools.common.jdt.debug/3.16.0.v20210224-1041/org.jboss.tools.common.jdt.debug-3.16.0.v20210224-1041.jar) to method sun.jvmstat.monitor.MonitoredHost.getMonitoredHost(java.lang.String)
2021-05-18T13:30:30.9112589Z WARNING: Please consider reporting this to the maintainers of org.jboss.tools.common.jdt.debug.tools.internal.Tools
2021-05-18T13:30:30.9115904Z WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
2021-05-18T13:30:30.9125459Z WARNING: All illegal access operations will be denied in a future release
2021-05-18T13:30:31.6763833Z SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
2021-05-18T13:30:31.6768354Z SLF4J: Defaulting to no-operation (NOP) logger implementation
2021-05-18T13:30:31.6770652Z SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
2021-05-18T13:30:35.0480807Z SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
2021-05-18T13:30:35.0525066Z SLF4J: Defaulting to no-operation (NOP) logger implementation
2021-05-18T13:30:35.0526382Z SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
2021-05-18T14:10:16.0817548Z [ERROR] Timeout 2400 s exceeded. Process was killed.
```

I guess there is a popup

**Comment by apupier** [comment]
moved away from guthub as CI

**Pull request closed without merging**